x-pack/plugins/endpoint/public/applications/endpoint/store/policy_details/selectors.ts

```diff
@@ -32,14 +33,16 @@ export const policyIdFromParams: (state: PolicyDetailsState) => string = createS
   }
 );
 
+const defaultFullPolicy: Immutable<PolicyConfig> = policyConfigFactory();
+
 /**
  * Returns the full Endpoint Policy, which will include private settings not shown on the UI.
  * Note: this will return a default full policy if the `policyItem` is `undefined`
  */
-export const fullPolicy: (s: PolicyDetailsState) => PolicyConfig = createSelector(
+export const fullPolicy: (s: Immutable<PolicyDetailsState>) => PolicyConfig = createSelector(
   policyDetails,
   policyData => {
-    return policyData?.inputs[0]?.config?.policy?.value ?? generatePolicy();
+    return policyData?.inputs[0]?.config?.policy?.value ?? defaultFullPolicy;
```

Comment:
any time we return a default policy, return the same (immutable) object reference